### PR TITLE
Add ITV.com

### PR DESCRIPTION
**Pull request by kymckay**
Another simple entry

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
The link doesn't seem to be a direct deletion one. Does the page have a very hard to miss delete button or similar?

Otherwise it might be useful to introduce some notes.

File: _data/sites.json [hunk 1]
```diff
@@ -5419,6 +5419,15 @@
         ]
     },
 
+    {
+        "name": "ITV",
+        "url": "https://www.itv.com/hub/user/profile",
```
Comment (Spectroxx) [inline _data/sites.json:5424]: Add notes so the directive is more concise with the end-user.

**Review by Spectroxx — CHANGES_REQUESTED** [review]
Notes should be required on all pull requests for sites.json
Specifically when adding sites.

**Comment by kymckay** [comment]
I disagree since default notes were added. However, for this entry I will add a short note.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**